src/shared/components/post/post-listing.tsx

```diff
@@ -335,13 +335,6 @@ export class PostListing extends Component<PostListingProps, PostListingState> {
               {i18n.t("bot_account").toLowerCase()}
             </span>
           )}
-          {(post_view.creator_banned_from_community ||
-            isBanned(post_view.creator)) && (
-            <span className="mx-1 badge badge-danger">{i18n.t("banned")}</span>
-          )}
-          {post_view.creator_blocked && (
-            <span className="mx-1 badge badge-danger">{"blocked"}</span>
-          )}
```

Comment:
iirc if its a single post fetch (IE not GetPosts, but GetPost), meaning you know the URL, you can still see it, even if you do have them blocked, because the back-end doesn't care to filter there. But yes this probably isn't necessary.